graph-manager-docs/source/schema-registry.mdx

```diff
@@ -33,15 +81,27 @@ id      schema        tag
 190330  example-4218  current
 ```
 
-The CLI fetches your schema via a configuration option you set up in [Prerequisites](#prerequisites) and pushes it to Graph Manager.
+The CLI fetches your schema via a configuration option you set up in the project configuration above and pushes it to Graph Manager.
+
+### Registering manually to a variant
 
-Every time you push a new version of your schema, it's appended to your graph's schema history.
+To register a schema to a particular [variant of your graph](#managing-environments-with-variants), include the `--variant=<VARIANT>` flag in your `apollo service:push` command, like so:
+
+```bash
+apollo service:push --variant=beta
+```
 
-## Registering a schema via continuous delivery
+If you omit the `--variant` flag, the `apollo service:push` command pushes to the default graph variant, named `current`.
 
-To get the most out of Graph Manager, you should **register each update to your production schema as soon as it occurs**. To achieve this, include a call to `apollo service:push` in your continuous delivery pipeline.
+## Registering with continuous delivery
 
-Here is a sample continuous delivery configuration for pushing a schema to Apollo using CircleCI:
+To get the most out of Graph Manager, you should **register each update to your production schema as soon as it occurs**. Consequently, schema registration should be part of your continuous delivery pipeline.
+
+If you're using [automatic schema reporting](#automatic-registration), registration happens automatically each time you deploy your production server. No additional continuous delivery configuration is required.
```

Comment:
should we link to the "tie-breaking" rules here, and explain that the registration will happen at the _end_ of the deploy rather than the beginning?

Reply:
Ahh, I see that section didn't carry over to these docs. I think it's important to document what users should expect to happen when they deploy, since there will be a slight delay in upgrading traffic. It should have had its own section in the preview docs, but it's here: https://github.com/apollographql/apollo-schema-reporting-preview-docs#the-schema-reporting-endpoint

Reply:
Do you think it's also worthwhile to note how schema reporting interacts with calls to `service:push` or is it unnecessary to document what we might consider the "expected" behavior?

Reply:
One last section that is missing is the additional config option for _which schema_ we send and how to override it: https://github.com/apollographql/apollo-schema-reporting-preview-docs#the-schema-reporting-endpoint

Reply:
Whoops! Yep, straight-up left stuff out